server/main.lua

```diff
@@ -4,7 +4,14 @@ TriggerEvent('esx:getSharedObject', function(obj) ESX = obj end)
 
 RegisterNetEvent('blarglebus:finishRoute')
 AddEventHandler('blarglebus:finishRoute', function(amount)
+    local xPlayer = ESX.GetPlayerFromId(source)
+    
+    if xPlayer.job.name == 'busdriver' then
     ESX.GetPlayerFromId(source).addMoney(amount)
```

Comment:
Since you're calling GetPlayerFromId on line 7 and storing it, I believe this line can be updated to the following, so the player isn't looked up twice:
```
        xPlayer.addMoney(amount)
```